docs/getting-started/ali-function-compute.md

```diff
@@ -0,0 +1,157 @@
+
+```sh
+npm install @serverless-devs/s -g
+```
+
+Add the AK & SK configuration
+
+```sh
+s config add 
+# Please select a provider: Alibaba Cloud (alibaba)
+# Input your AccessKeyID & AccessKeySecret
+```
+
+## 2. Hello World
+
+Edit `src/index.ts`:
+
+```ts
+import { serve } from "@hono/node-server";
+import { Hono } from "hono";
+
+const REQUEST_ID_HEADER = "x-fc-request-id";
+
+const app = new Hono();
+
+app.post("/initialize", (c) => {
+  console.log(`RequestId: ${c.req.header(REQUEST_ID_HEADER)}`);
+  return c.text("Initialize");
+});
+
+app.post("/invoke", (c) => {
+  console.log(`RequestId: ${c.req.header(REQUEST_ID_HEADER)}`);
+  return c.text("Invoke");
+});
+
+app.get("/", (c) => {
+  return c.text("Hello from index!");
+});
+
+app.get("/hello", (c) => {
+  return c.text("Hi!");
+});
+
+const port = 9000;
+console.log(`Server is running on port ${port}`);
+
+serve({
+  fetch: app.fetch,
+  port,
+});
+```
+
+Edit `package.json`:
+
+```json
+{
+  "scripts": {
+    "build": "esbuild --bundle --outfile=./dist/index.js --platform=node --target=node20 ./src/index.ts",
+    "dev": "node ./dist/index.js",
+    "deploy": "s deploy -y"
+  },
+  "dependencies": {
+    "esbuild": "^0.21.4",
+    "@hono/node-server": "^1.11.2",
+    "hono": "^4.4.3"
+  },
+  "devDependencies": {
+    "@serverless-devs/s": "^3.0.8"
+  }
+}
+```
+
+Edit `tsconfig.json`
+
+```json
+{
+  "compilerOptions": {
+    "target": "ESNext",
+    "module": "ESNext",
+    "moduleResolution": "Bundler",
+    "strict": true,
+    "skipLibCheck": true,
+    "lib": [
+      "ESNext"
+    ],
+    "types": [],
+    "jsx": "react-jsx",
+    "jsxImportSource": "hono/jsx"
+  },
+}
+```
+
+Edit `s.yaml`:
+
+```yaml
+edition: 3.0.0
+name: my-app
+access: "default"
+
+vars:
+  region: "us-west-1"
+
+resources:
+  my_app:
+    component: fc3
+    props:
+      region: ${vars.region}              
+      functionName: "my-app"
+      runtime: "custom.debian10"
+      description: 'hello world by Hono'
+      timeout: 10
+      memorySize: 512
+      environmentVariables:
+        PATH: >-
+          /var/fc/lang/nodejs20/bin:/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin
+        NODE_PATH: /opt/nodejs/node_modules
+      cpu: 0.5
+      diskSize: 512
+      code: ./dist
+      customRuntimeConfig:
+        command:
+          - node
+          - index.js
+        port: 9000
+      triggers:
+        - triggerConfig:
+            methods:
+              - GET
+              - POST
+              - PUT
+              - DELETE
+            authType: anonymous
+            disableURLInternet: false
+          triggerName: default
+          description: ''
+          qualifier: LATEST
+          triggerType: http
+```
+
+## 3. Deploy
+
+Finally, run the command to deploy:
+
+```sh
+npm run install # install dependencies
+npm run build # Compile the TypeScript code to JavaScript
+s deploy # Deploy the function to Alibaba Cloud Function Compute
```

Comment:
Shouldn't this be `npm run deploy` to run the deploy script added?

Reply:
fixed by <https://github.com/honojs/website/pull/417/commits/936c003e3e418f4120812f91a468a32a9168ba13>